migrate/v0_26/iavl-v1.md

```diff
@@ -9,6 +9,8 @@ unless otherwise specified.
 
 Steps for using IAVL V1:
 
+**Note:** Nodes running IAVL V1 should set `iavl-disable-fastnode = true` in their `app.toml` file.
```

Comment:
Should we make it more visible? With [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

```suggestion
> [!IMPORTANT]  
> Nodes running IAVL V1 should set `iavl-disable-fastnode = true` in their `app.toml` file.
```